packages/common/src/doctype.ts

```diff
@@ -70,6 +70,10 @@ export interface DocNext {
     metadata?: DocMetadata;
 }
 
+export interface LogEntry {
```

Comment:
Should we call this LogRecord?  We seem to call these things Records everywhere else.

Reply:
I think we then risk overloading the term "Record" a bit. Usually we use the term to refer to the content of the record that is stored in IPFS. In this log entry we're just storing the CID of the record + metadata.